### PR TITLE
File locations in commandline build script

### PR DESCRIPTION
**Pull request by Loadus**
Slightly heavy modification to allow the commandline build script to find the required files. I would love a solution where this information is passed on to that script via either asking Processing or a settings file etc. 

In this version, the user has to input the locations, it's a bit meh.

### CODE REVIEW

**Comment by codeanticode** [comment]
@Loadus Thanks a lot for your pull requests! I will review and merge asap!

File: processing-android [hunk 1]
```diff
@@ -15,6 +15,11 @@
 
 # Thanks to Ferdinand Kasper for this build script. [fry]
 
+# Target version
+ANDROID_VERSION="17"
+
+# Your sketchbook location
+SKETCHBOOK_LOCATION="/path/to/your/sketchbook"
```
Comment (codeanticode) [inline processing-android]: ANDROID_VERSION and SKETCHBOOK_LOCATION can be read from Processing's preferences file, which a specific location for each OS (for example ~/Library/Processing/preferences.txt on Mac OS X).

Comment (Loadus) [inline processing-android]: @codeanticode I'll cook up an alternative version where this info is read from the preferences.txt, momento. I'll also look into creating build scripts for OSX + win32 (I might not know how to do those).

**Comment by Loadus** [comment]
@codeanticode Automagical ---> https://github.com/processing/processing-android/pull/170

**Comment by codeanticode** [comment]
@Loadus is this one superseded by PR https://github.com/processing/processing-android/pull/170 as well?

**Pull request closed without merging**